charts/thoras/templates/monitor/deployment.yaml

```diff
@@ -78,5 +78,7 @@ spec:
           - name: SLACK_WORKSPACE_ID
             value: {{ .Values.thorasMonitor.slackWorkspaceID }}
           - name: SLACK_CHANNEL_ID
-            value: {{ .Values.thorasMonitor.slackChannelID }}
+            value: {{ .Values.thorasMonitor.slackChannelID }}\
```

Comment:
This an accidental backslash?

Reply:
Thank you for catching that! I have removed it.